src/Microsoft.IdentityModel.Abstractions/DataClassification.cs

```diff
@@ -0,0 +1,82 @@
+    /// </summary>
+    /// <remarks>
+    /// See http://aka.ms/AssetClassificationStandard for more details about the asset classification standard.
+    /// This is meant to identify the most consequential classification level (e.g. if an entry contains public data
+    /// and EUII, it should be classified as EUII).
+    /// </remarks>
+    public enum DataClassification
+    {
+        /// <summary>
+        /// Data used to manage access to administrative roles or sensitive functions.
+        /// </summary>
+        AccessControlData = 0,
+
+        /// <summary>
+        ///This is the data, information and code that C1 admins, C1 users, and C2 users (if present) provide to, transfers in, stores in or
+        /// process in the Microsoft online service or product.
+        /// </summary>
+        /// <remarks>The term C1 refers to Microsoft’s customer.The term C2 refers to the customer’s customer.</remarks>
+        CustomerContent = 10,
+
+        /// <summary>
+        /// End User Identifiable Information.
+        /// Data that directly identifies or could be used to identify the authenticated C1 or C2 user of a Microsoft service. EUII does not contain C1 admin data.
+        /// EUII does not extend to other personal information found in Customer Content (e.g. contact lists are Customer Content), contact lists are Customer Content). 
+        /// </summary>
+        /// <remarks>The term C1 refers to Microsoft’s customer.The term C2 refers to the customer’s customer.</remarks> 
+        EUII = 20,
+
+        /// <summary>
+        /// All data provided to Microsoft and generated by Microsoft as part of Support activities.
+        /// </summary>
+        /// <remarks>The term C1 refers to Microsoft’s customer.The term C2 refers to the customer’s customer.</remarks>  
+        SupportData = 30,
+
+        /// <summary>
+        /// Contact and billing/purchase/payment/license information for the C1 enterprise,  including the C1 admin and any C1 subdelegated admins.
+        /// </summary>
+        /// <remarks>The term C1 refers to Microsoft’s customer.The term C2 refers to the customer’s customer.</remarks>  
+        AccountData = 40,
+
+        /// <summary>
+        /// Publicly available personal information that Microsoft obtains from external sources. Public Personal Data is
+        /// not <see cref="CustomerContent"/>, <see cref="EUII"/>, or <see cref="EUPI"/> since the data was not input.
+        /// </summary>
+        PublicPersonalData = 50,
+
+        /// <summary>
+        /// End User Pseudonymous Identifiers.
+        /// An identifier created by Microsoft tied to the C1 or C2 user of a Microsoft service. When EUPI is
+        /// combined with other information (such as a mapping table), it identifies the end user. 
+        /// </summary>
+        /// <remarks>The term C1 refers to Microsoft’s customer.The term C2 refers to the customer’s customer.</remarks>   
```

Comment:
Example would be helpful.

Reply:
added examples for each from original doc